e2e/src/usecases/RestoreAccountOnboarding.js

```diff
@@ -63,9 +58,15 @@ export default RestoreAccountOnboarding = () => {
       await scrollIntoView('Restore', 'ImportWalletKeyboardAwareScrollView')
       await element(by.id('ImportWalletButton')).tap()
 
-      // verification step comes after restoring wallet, skip this step
-      await waitForElementId('PhoneVerificationSkipHeader')
-      await element(by.id('PhoneVerificationSkipHeader')).tap()
+      try {
```

Comment:
instead of a try catch, can we explicitly define the expected flow (testID) for each variant of the test? It seems like this implementation is open to accidentally passing a test.

Reply:
As I understand it, if the try block fails, the main assertions below, `// verify that the correct account was restored`, will still be tested. I do not see how we'd be at risk of this test accidentally passing in its current state.

Reply:
we do know already if the 2 accounts tested here require phone verification or whether the balance is 0 right? Could we just make that explicit. I guess scenarios where this pass accidentally is if account has 0 balance but the confirm dialog doesn't show up, or if phone verification is required, but is never shown. Maybe those are not critical to test, but seems like it is something we could define explicitly (e.g., add a boolean in the it.each for whether phone verification / 0 balance dialog should show up and include the assertion)